README.md

```diff
@@ -247,9 +246,8 @@ Sent from the rover to inform Mission Control of the rover's current position in
 - `orientX` - refers to the orientation quaternion X component
 - `orientY` - refers to the orientation quaternion Y component
 - `orientZ` - refers to the orientation quaternion Z component
-- `posX` - refers to the X position of the rover in world reference frame in meters
-- `posY` - refers to the Y position of the rover in world reference frame in meters
-- `posZ` - refers to the Z position of the rover in world reference frame in meters
+- `lon` - refers to the longitude of the rover in world reference frame in degrees, minutes, seconds
+- `lat` - refers to the latitude of the rover in world reference frame in degrees, minutes, seconds
```

Comment:
The units here are actually just degrees in floating point, we don't have separate representations for minutes and seconds.